man/rgbasm.5

```diff
@@ -324,9 +343,41 @@ with a zero constant as either operand will be constant 0, and
 .Sq ||
 with a non-zero constant as either operand will be constant 1, even if the other operand is non-constant.
 .Pp
-.Ic \&!
+.Sq \&!
 returns 1 if the operand was 0, and 0 otherwise.
 Even a non-constant operand with any non-zero bits will return 0.
+.Ss Integer functions
+Besides operators, there are also some functions which have more specialized uses.
+.EQ
+delim $$
+.EN
+.Bl -column "BITWIDTH(n)"
+.It Sy Name Ta Sy Operation
+.It Fn HIGH n Ta Returns the top 8 bits of
+.Ar n .
+.It Fn LOW n Ta Returns the bottom 8 bits of
+.Ar n .
+The result may be constant if
+.Nm
+is able to compute it.
```

Comment:
```suggestion
is able to compute it (according to the same rules as the 
.Sq &
operator).
```

Reply:
"The result may be constant if .Nm is able to compute it" occurs three times here. We don't spell out the internals of how RGBASM is determining that every time.

Reply:
Since those rules are outlined elsewhere, I think it's worth pointing them out explicitly.

Alternatively, if `HIGH` and `LOW` are documented as the convenience shortcuts that they *really* are, then this point becomes moot.